### ISSUE TITLE
[Request] Reviewing VS Tools for Tizen 

### ISSUE THREAD

**Issue by journey2w** [Samsung/tizen-docs#46]
안녕하세요 김소영 차장님.

 VS Tools for Tizen 1.0이 10월 말 릴리즈에 포함됩니다.

그래서 문서를 1.0 에 맞게 담당자께서 작성해 주셨는데요, 사내 GitHub을 사용하고 계셔서 Markdown 로 작성하셨습니다.

그래서 저희한테 영문 검수가 필요한 파일을 주셨는데, tizen-docs github에 넣었습니다. MD 파일을 그대로 영문 검수해주시면 이 파일을 VS Tools 사내 GitHub에도 복사하고 tizen-docs 에 나중에 그대로 사용하고, TD에는 MD 파일을 HTML로 editing 작업을 하도록 하겠습니다.

아래 문서에 대해 영문 검수만 부탁드립니다. (tizen-docs를 공식적으로 사용하는 시점에 문서 path와 directory path는 맞추도록 하겠습니다.)

작업하셔야할 branch는 vs-tools-tizen 입니다.

제목 | developer tizen 문서 위치 | 신규 문서 위치
-- | -- | --
Installing Visual Studio Tools for Tizen | https://developer.tizen.org/development/tizen-.net-preview/getting-started/installing-visual-studio-tools-tizen | tizen-docs/docs/vs-tools-tizen/how-to-install.md
Project Wizard | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/project-wizard | tizen-docs/docs/vs-tools-tizen/tools/project-wizard.md
Certificate Manager | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/certificate-manager | tizen-docs/docs/vs-tools-tizen/tools/certificate-manager.md
Tizen Manifest Editor | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/tizen-manifest-editor | tizen-docs/docs/vs-tools-tizen/tools/manifest-editor.md
Emulator Manager | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/emulator-manager | tizen-docs/docs/vs-tools-tizen/tools/emulator-manager.md
Emulator Control Panel | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/emulator-control-panel | 삭제 (VS Tools 쪽 추가 피쳐 없음)
API and Privilege Checker | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/api-and-privilege-checker | tizen-docs/docs/vs-tools-tizen/tools/api-and-privilege-checker.md
Device Manager | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/device-manager | tizen-docs/docs/vs-tools-tizen/tools/device-manager.md
Log Viewer | https://developer.tizen.org/development/tizen-.net-preview/visual-studio-tools-tizen/log-viewer | tizen-docs/docs/vs-tools-tizen/tools/log-viewer.md
Profiler | 신규 피쳐 | tizen-docs/docs/vs-tools-tizen/tools/profiler-user-manual.md

**Comment by journey2w** [comment]
Written by KIm, Soyoung.
===============================================================
안녕하세요. 강영애님,


전달해 주신 VS Tools for Tizen(https://github.com/Samsung/tizen-docs/issues/46) 문서에 대해서는 검수가 완료가 되는데로 GitHub에 리뷰 올릴 예정입니다. 

일단 아래 리스트 해 주신 문서에 대해서는 검수를 하였는데요. 기존에 있었던 VS tools section 의 문서들 중에 first app topic, 2가지 tool 관련 토픽, packaging 문서 전체는 빠진 것 같습니다. 해당 문서들은 이후 진행 예정인지요 아니면 완전히 삭제가 된 것인지 문의 드립니다. 
 

감사합니다.

김소영 드림

**Comment by journey2w** [comment]
I think you are using Internet Explorer that is not compatible with GitHub. Please use Chrome or Firefox.
You can see some files on https://github.com/Samsung/tizen-docs/tree/vs-tools-tizen/docs/vs-tools-tizen/, docs/vs-tools-tizen/packaging/, and docs/vs-tools-tizen/tools/

**Comment by l10nbridge** [comment]
Before we push any edited content in, could you please clarify a few confusing issues about the VS Tool content?

1. What kind of templates are actually available in the Project Wizard in the latest VS Tools version?
The current topics are quite confusing, because there are clearly 2 different versions described, and we do not know which one is the latest.
Some topics describe a setup where the project wizard has 3 templates in the “Tizen” section of the wizard side menu. 
Other topics describe a setup where the project wizard has the “Cross-Platform” and “Native” sections under the “Tizen” section, with a single template in the “Cross-Platform “ section, and 2 templates in the “Native“ section.
Which one is correct? (We need to modify multiple topics based on this answer…)

2. The packaging folder contains the how-to-create-tpk.md file, which describes how you can build a project with .NET CLI and Visual Studio.
In addition, there are 2 separate files (how-to-build-cli.md and how-to-build-vs.md), which seem to cover the exact same instructions, with just slightly more detail.
Why exactly are all these topics needed? Could we gather all the details into the how-to-create-tpk.md file, and delete the other two? Or, if you want to keep the CLI and VS instructions separate, could we delete the how-to-create-tpk.md file?

3. All the images are in an "image" folder, instead of in folder-specific "media" folders. Is that OK?

4. Several images have font issues (they show the strikethrough W character in file paths). Also, some Visual Studio images are using a black background theme. It would be better to take all screenshots with the same Windows 10 theme.
What are we going to do with the images? If we do not have access to the latest tool version, we cannot fix these. Are you going to look into this, or are the images going to be left as they are? Or are we going to ignore this issue until after the release, at which point we have access to the latest tool and can retake all images, as needed?

5. Several files in the "packaging" directory contain empty headings at the end of the topics. Since there is no content, can we remove those headings? Or is the content available somewhere where we could copy it from?

**Comment by yhshin217** [comment]
Hello, @l10nbridge members.
Let me answer about your question.

> 1. What kind of templates are actually available in the Project Wizard in the latest VS Tools version?
The current topics are quite confusing, because there are clearly 2 different versions described, and we do not know which one is the latest.
Some topics describe a setup where the project wizard has 3 templates in the “Tizen” section of the wizard side menu.
Other topics describe a setup where the project wizard has the “Cross-Platform” and “Native” sections under the “Tizen” section, with a single template in the “Cross-Platform “ section, and 2 templates in the “Native“ section.
Which one is correct? (We need to modify multiple topics based on this answer…)

There are 4 templates for Tizen .NET development. 
![image](https://user-images.githubusercontent.com/16409430/31698892-c202addc-b3fa-11e7-81f7-772457349956.png)
```
Tizen 
     - Cross-Platform
            - Blank App (Xamarin.Forms)
     - Native
            - Blank App (ElmSharp-Beta)
            - Blank App (Tizen.NUI)
     - Class Library (.NET Standard)  
```

> 2. The packaging folder contains the how-to-create-tpk.md file, which describes how you can build a project with .NET CLI and Visual Studio.
In addition, there are 2 separate files (how-to-build-cli.md and how-to-build-vs.md), which seem to cover the exact same instructions, with just slightly more detail.
Why exactly are all these topics needed? Could we gather all the details into the how-to-create-tpk.md file, and delete the other two? Or, if you want to keep the CLI and VS instructions separate, could we delete the how-to-create-tpk.md file?

For editing easily in development stage, we divided them. In case of official stage, you can combine them. Actually, it's better to understand in developer side.

> 3. All the images are in an "image" folder, instead of in folder-specific "media" folders. Is that OK?
@journey2w , could you confirm about this question?

> 4. Several images have font issues (they show the strikethrough W character in file paths). Also, some Visual Studio images are using a black background theme. It would be better to take all screenshots with the same Windows 10 theme.
What are we going to do with the images? If we do not have access to the latest tool version, we cannot fix these. Are you going to look into this, or are the images going to be left as they are? Or are we going to ignore this issue until after the release, at which point we have access to the latest tool and can retake all images, as needed?

Could you list up the images which should be updated? If let us know them, we will make new shots.

> 5. Several files in the "packaging" directory contain empty headings at the end of the topics. Since there is no content, can we remove those headings? Or is the content available somewhere where we could copy it from?

Sure, you can remove them. And, if you update what you removed, we will update it in our site too.

**Comment by l10nbridge** [comment]
Related to issue 4 about images:

Could you please take new screenshots for the following images:

- CPS_exclude_reference.png (black theme)
- CPS_exclude_reference_disable.png (black theme)
- CPS_new_project.png (black theme)
- dm_explorer.png (Korean characters)
- HowToBuild_AddNugetFeed.png (Strikethrough W characters in file paths instead of “\”)
- HowToBuild_NewProject.png (Strikethrough W characters in file paths instead of “\”)
- HowToInstall_BaselineInstall6.png (Strikethrough W characters in file paths instead of “\”)
- HowToInstall_CheckToolPath.png (Strikethrough W characters in file paths instead of “\”)
- nuget-ref-tizen.net.sdk.png (black theme)
- nuget-tizen.net.sdk.png (black theme)
- profiler_output_window.png (Strikethrough W characters in file paths instead of “\”)
- ProjectWizard_CrossNew.png (Strikethrough W characters in file paths instead of “\”)
- ProjectWizard_NativeNew.png (Strikethrough W characters in file paths instead of “\”)
- ProjectWizard_Profile.png (Strikethrough W characters in file paths instead of “\”)
- SamsungExt_option.png (Strikethrough W characters in file paths instead of “\”)

Also, can you confirm whether the "HowToBuild_RefTizenNETSdk.png" screenshot is up-to-date? 
Is the project name really “…Tizen.Mobile”, and not “…TizenMobile”? If the name is wrong, the “manifest_overview.png”, “manifest_text_editor.png”, “mouse_right.png”, “ProjectWizard_Solution.png”, “session_explorer.png”, and “session_properties.png” files also need updating.

**Comment by journey2w** [comment]
All the images are in an "image" folder, instead of in folder-specific "media" folders. Is that OK?
=> "media" is good for this repo, "image" is good for VS tools repo (not existed here), or "images" is good for TD.

Regardless of what you choose, if you choose one, you need to modify the other two.

So...I'd like to choose "media".

**Comment by yhshin217** [comment]
@l10nbridge , Sorry for late answer.

> CPS_exclude_reference.png (black theme)
CPS_exclude_reference_disable.png (black theme)
dm_explorer.png (Korean characters)
HowToBuild_AddNugetFeed.png (Strikethrough W characters in file paths instead of “\”)
HowToBuild_NewProject.png (Strikethrough W characters in file paths instead of “\”)
HowToInstall_BaselineInstall6.png (Strikethrough W characters in file paths instead of “\”)
HowToInstall_CheckToolPath.png (Strikethrough W characters in file paths instead of “\”)
nuget-tizen.net.sdk.png (black theme)
profiler_output_window.png (Strikethrough W characters in file paths instead of “\”)
ProjectWizard_CrossNew.png (Strikethrough W characters in file paths instead of “\”)
ProjectWizard_NativeNew.png (Strikethrough W characters in file paths instead of “\”)
ProjectWizard_Profile.png (Strikethrough W characters in file paths instead of “\”)
SamsungExt_option.png (Strikethrough W characters in file paths instead of “\”)

Updated images were applied with https://github.sec.samsung.net/dotnet/vs-tools-cps/commit/66ca246bd5cf0866e1c169f2c2818965270faf33

> Also, can you confirm whether the "HowToBuild_RefTizenNETSdk.png" screenshot is up-to-date?
Is the project name really “…Tizen.Mobile”, and not “…TizenMobile”? If the name is wrong, the “manifest_overview.png”, “manifest_text_editor.png”, “mouse_right.png”, “ProjectWizard_Solution.png”, “session_explorer.png”, and “session_properties.png” files also need updating.

.Tizen.Mobile , .Tizen.Wearable , .Tizen.TV are right.

**Comment by journey2w** [comment]
@l10nbridge
I sent Ms. Soyoung Kim all files changed by https://github.sec.samsung.net/dotnet/vs-tools-cps/commit/66ca246bd5cf0866e1c169f2c2818965270faf33 commit.

I wanted to push a new commit to your porked github but I couldn't. You changed the filenames of images so I didn't know exact which ones are the original files.
Thanks.

**Comment by l10nbridge** [comment]
@journey2w 
Thank you, we have received the files and will push them in as a new commit as soon as we have renamed them.

**Comment by journey2w** [comment]
Resolved: #62
